com.unity.formats.fbx/Editor/ExportSettings.cs

```diff
@@ -10,6 +10,43 @@
 using UnityEditor.Presets;
 
 namespace UnityEditor.Formats.Fbx.Exporter {
+
+    /// <summary>
+    /// Enum specifying the FBX export format options.
+    /// </summary>
+    public enum ExportFormat { ASCII = 0, Binary = 1 }
+
+    /// <summary>
+    /// Enum specifying the type of data to include in the export
+    /// (Model only, animation only, or model and animation).
+    /// </summary>
+    public enum Include { Model = 0, Anim = 1, ModelAndAnim = 2 }
```

Comment:
if we have `Model=1` `Anim=2` and `ModelAndAnim=3` then you can do `Include.Anim | Include.Model`